seed_services_client/identity_store.py

```diff
@@ -1,25 +1,24 @@
-from demands import JSONServiceClient
+from .seed_services import SeedServicesApiClient
 
 
-class IdentityStoreApiClient(object):
+class IdentityStoreApiClient(SeedServicesApiClient):
     """
     Client for Identity Store Service.
 
     :param str auth_token:
-
         An access token.
 
     :param str api_url:
         The full URL of the API.
 
+    :param JSONServiceClient session:
+        An instance of JSONServiceClient to use
+
     """
 
-    def __init__(self, auth_token, api_url, session=None):
-        headers = {'Authorization': 'Token ' + auth_token}
-        if session is None:
-            session = JSONServiceClient(url=api_url,
-                                        headers=headers)
-        self.session = session
+    def __init__(self, auth_token, api_url, session=None, **kwargs):
+        super(IdentityStoreApiClient, self).__init__(
+            auth_token, api_url, session=None, **kwargs)
```

Comment:
Also updated the first identity store test to make sure it doesn't happen again.